pages/linux/kdocker.md

```diff
@@ -0,0 +1,28 @@
+# kdocker
+
+> Easily dock applications to system tray.
```

Comment:
```suggestion
> Easily dock applications to the system tray.
```

Reply:
@navarroaxel your a Linux user, what do you think about this

Reply:
I think the suggestion from @bl-ue is correct, the sentence needs the `the`.